health-sdk/sdk/src/main/java/net/gini/android/health/sdk/moreinformation/MoreInformationFragment.kt

```diff
@@ -0,0 +1,201 @@
+package net.gini.android.health.sdk.moreinformation
```

Comment:
[Request] Please use the background color from the design for the `MoreInformationFragment`. Currently the background color looks different in light mode and dark mode:

Light mode:
![Screenshot_20240227_171713](https://github.com/gini/gini-mobile-android/assets/13149498/fc9f731d-62fc-45dd-ab1f-8f20b2cddbc5)

Dark mode:
![Screenshot_20240227_171654](https://github.com/gini/gini-mobile-android/assets/13149498/56796d29-817f-4168-a678-e4baa46f0054)

Reply:
I did change the background to the color in the design: `ghs_dark_07`.

However, I see we do not have all the colors defined in `colors.xml(night)`. As a result, it currently looks like this when switching to dark mode: 

Perhaps we need to define the colors for dark mode as well?

![Screenshot_20240228_093629_Gini Health](https://github.com/gini/gini-mobile-android/assets/5913066/76341f60-be8d-4215-ab0a-b4e9730b0760)